modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -39,15 +39,15 @@ spec:
           pod: {}
         - name: bridge-net <1>
           multus:
-            networkName: <a-bridge-network> <2>
+            networkName: <a-bridge-network/network-namespace> <2>
 ...
 ----
 <1> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
 <2> The name of the network attachment definition.
 +
 [NOTE]
 ====
-The example virtual machine is connected to both the `default` pod network and `bridge-net`, which is defined by a network attachment definition named `a-bridge-network`.
+The example virtual machine is connected to both the `default` pod network and `bridge-net`, which is defined by a network attachment definition named `a-bridge-network`. However, if the network is not in the same namespace as the VM, you can use a combination of the network and the namespace as the name of the network attachment definition.
```

Comment:
Maybe we should reference "network attachment definition" explicitly instead of just "network", so it cannot be confused with the pod network.

And the "combination" should be now switched